test/api/groups.coffee

```diff
@@ -163,6 +262,50 @@ describe "Groups", ->
         expect(party.memberCount).to.equal group.memberCount
         done()
 
+    it "prevents user from joining or creating a second party", (done) ->
+      request.post(baseURL + "/groups").send(
+        name: "TestGroup"
+        type: "party"
+      ).end (res) ->
+        expectCode res, 400
+        expect(res.body.err).to.equal "Already in a party, try refreshing."
+        done()
+
+    context "joining a party", ->
+
+      it "prevents user from joining a party when they haven't been invited", (done) ->
+        registerNewUser (err, user) ->
+          request.post(baseURL + "/groups/" + group._id + "/join").send(
+              name: "TestGroup"
+              type: "guild",
+          )
+          .set("X-API-User", user._id)
+          .set("X-API-Key", user.apiToken)
+          .end (res) ->
+            expectCode res, 401
+            done()
+        , false
+
+      it "allows users to join a party when they have been invited", (done) ->
+        tmpUser = undefined
+        registerNewUser (err, user) ->
+          tmpUser = user
+          inviteURL = baseURL + "/groups/" + group._id + "/invite"
+          request.post(inviteURL).send(
+            uuids: tmpUser._id
+          )
+          .set("X-API-User", tmpUser._id)
+          .set("X-API-Key", tmpUser.apiToken)
+          .end ->
+            request.post(baseURL + "/groups/" + group._id + "/join")
+            .set("X-API-User", tmpUser._id)
+            .set("X-API-Key", tmpUser.apiToken)
+            .end (res) ->
+              expectCode res, 401
+              expect(res.body.err).to.equal "Can't join a group you're not invited to."
```

Comment:
Why is this expectation in the "Allow users to join a party when they have been invited"?